### DISCUSSION TITLE
Multiple waste same calendar

### DISCUSSION THREAD

**Discussion by alexdelli** [mampfes/hacs_waste_collection_schedule#1711] [Q&A] [answered]
Hi, could you help me with this calendar?
I have this kind of information in my calendar:
Castelfranco Veneto - fuori centro storico: Carta, Umido
Castelfranco Veneto - fuori centro storico: Umido
Castelfranco Veneto - fuori centro storico: Carta

I create different type and sensors in this way and it works:
- type: "Castelfranco Veneto - fuori centro storico: Carta, Umido"
          alias: Carta e Umido
- type: "Castelfranco Veneto - fuori centro storico: Umido"
          alias: Umido
- type: "Castelfranco Veneto - fuori centro storico: Carta"
          alias: Carta

But I want to create only 2 different waste type: Carta and Umido and not a separate day for Carta, Umido 
Is it possible to create this type of waste?
thank you

**Comment by 5ila5** [comment, marked as answer]
resolved by #1715

> you can use the regex and split_at parameter to clean the name and split them at a given separator I added description for this in #1715 you can now find it in the description (https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/ics/contarina_it.md)
- https://github.com/mampfes/hacs_waste_collection_schedule/pull/1569#issuecomment-1906271554

**Reply by alexdelli** [reply]
thank you!
